avm/ptn/authorization/role-assignment/tests/e2e/sub.common/main.test.bicep

```diff
@@ -62,5 +62,6 @@ module testDeployment '../../../main.bicep' = {
     principalType: 'ServicePrincipal'
     location: resourceLocation
     subscriptionId: subscriptionId
+    resourceGroupName: resourceGroupName
```

Comment:
Why the resourceGroupName? Does it then not apply the roleAssignment to the RG scope, as opposed to the subscription scope that we want to test?